src/lib/dialog/dialog.spec.ts

```diff
@@ -405,10 +403,11 @@ describe('MdDialog', () => {
     });
 
     it('should default to null if no data is passed', () => {
-      let dialogRef: MdDialogRef<DialogWithInjectedData>;
-
-      expect(() => dialogRef = dialog.open(DialogWithInjectedData)).not.toThrow();
-      expect(dialogRef.componentInstance.data).toBeNull();
+      // TODO: double check that this does what i expect
```

Comment:
Ops, forgot to follow up on this one. I wanted to check that the `expect` inside the `expect.not.toThrow` won't cause any issues. I'll check it and refactor or remove the todo.